draft-ietf-core-coral.md

```diff
@@ -165,6 +167,60 @@ In contrast to RDF, however, the focus of CoRAL is not on the
 description of a graph of resources, but on the discovery of possible
 future application states.
 
+-->
+### Definitions
+
+The *CoRAL information model* is similar to the [Resource Description Framework (RDF)](#W3C.REC-rdf11-concepts-20140225) information model:
+Data is expressed as an (unordered) set of triples (also called statements),
+consisting of a subject, a predicate and an object.
+The predicate is always a URI in syntax normalized form, and subject and object are either a URI in syntax normalized form, or a concrete CBOR object (a literal).
+
+These triples form a directed graph with the subject and object being source and destination, and the predicate a description on the edge.
+That graph is equivalent to the data.
+
+To form a set and a graph, we define an equivalence relation:
+URIs are only equal to URIs and if they are identical byte-wise.
+Literals carry an identity from their serialization format (think of it as the file and cursor position from which it was read -- but if cbor-packed is involved it's more like the stack of cursors at parsing time),
+and are only equivalent to themselves.
+Triples are equivalent to each other if their subject, predicate and object are pair-wise equivalent.
+
+The *CoRAL structured information model* is an ordered tree across the graph of the CoRAL information model, whose root is the URI from which the document is obtained, and which may span edges multiple times.
+
+### Observations
+
+The structured form of a data set is in general not unique:
+If a node has more than one child, their sequence can be varied.
+If a node has more than one parent, its children may be expressed on any non-empty set of its parents
+to obtain a structured data set that expresses the same data set.
+
+Arbitrary data can not be expressed in a structured data set, because
+* There may not be a tree that covers the directed graph, or the tree's root may not be the URI from which the document is obtained.
+* There may be multiple edges into a literal, and the serialization can not build a file where these are expresed at the same spot.
+
+In particular, the precise data from one structured information document can only be expressed at the same URI.
+However, statements can be added to make a data set that is expressible elsewhere,
+and subsets of the data can be taken and expressed.
```

Comment:
there is probably a better way to say _"taken and expressed"_ but I don't have any concrete suggestion.